crates/gpui/src/platform/windows/window.rs

```diff
@@ -926,3 +1084,6 @@ unsafe fn set_window_long(hwnd: HWND, nindex: WINDOW_LONG_PTR_INDEX, dwnewlong:
         SetWindowLongW(hwnd, nindex, dwnewlong as i32) as isize
     }
 }
+
+const DRAGDROP_GET_COUNT: u32 = 0xFFFFFFFF;
```

Comment:
Where does this constant come from?

Reply:
Sorry for the delay, it was 2 am when I submitted this PR.

By [Microsoft](https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-dragqueryfilew):
> If the value of this parameter is 0xFFFFFFFF, DragQueryFile returns a count of the files dropped.